server/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "",
   "type":"module",
   "main": "server.js",
-"scripts": {
-    "reset": "node --env-file=config/.env.local config/reset.js",
-    "start": "nodemon --env-file=config/.env.local server.js"
+  "scripts": {
+    "reset": "node --env-file=../.env config/reset.js",
+    "start": "npm run reset && nodemon --env-file=../.env server.js"
```

Comment:
I'd recommend renaming your `.env` to `.env.local` and moving it into your `config` directory and updating the scripts to match 
```
"reset": "node --env-file=config/.env.local config/reset.js",
"start": "nodemon --env-file=config/.env.local server.js"
```

That way, we don't have to worry about having to make changes to our environment each time we review/devleop.